Create cross community webinar event (#51)
haam-community/haam-community.github.io: _posts/2024-05-08-event.md

---
layout: post

> title: "Cross-community #1"

Reviewer: ```suggestion
title: "Cross-community Webinar #1"
```

subtitle: "Improving reporting and reproducibility of lab work in biomolecular archaeology: saving your, your colleagues, and future you's time!"
categories: Events
tags: Cross-Community-Webinar
---

<div class="center" markdown="1" style="font-size:85%">

### At a glance:

> - **When:** 📅 _16th of May 2024 / 10-11h (CEST)_

Reviewer: ```suggestion
- **When:** 📅 _16th of May 2024 / 10:00-11:00 (CEST)_
```

- **Where:** 🌐 _Online_
- **Price:** 💰 _Free_
- **Target audience:** 🧑‍🔬🧑‍💻 Students and early career researchers

</div>

### 📣 We are excited to announce our first Cross-community Webinar! 📣

> [Dr. James A. Fellows Yates (MPI-EVA)](https://www.eva.mpg.de/archaeogenetics/staff/james-fellows-yates/) will be hosting our first cross-community webinar! Attendees will also have the opportunity to engage in a Q&A roundtable discussion with experts on [protocols.io](https://protocols.io/). This event is co-organized by the [AaRC (Animal aDNA Research Community)](https://animal-adna-research-community.github.io/AaRC.github.io/), **HAAM (Human Ancient DNA, Ancestry, and Mobility)**, [PAASTA (Palaeoproteomics and Archaeology, Society for Techniques and Advances)](https://paasta-community.github.io/), and [SPAAM (Standards, Precautions, and Advances in Ancient Metagenomics)](https://www.spaam-community.org/). It is open to all students and early career researchers.

Reviewer: ```suggestion
The four bioarchaeology communities are teaming up to announce the first Webinar in our Cross-Community series!
This event is co-organized by the [AaRC (Animal aDNA Research Community)](https://animal-adna-research-community.github.io/AaRC.github.io/), **HAAM (Human Ancient DNA, Ancestry, and Mobility)**, [PAASTA (Palaeoproteomics and Archaeology, Society for Techniques and Advances)](https://paasta-community.github.io/), and [SPAAM (Standards, Precautions, and Advances in Ancient Metagenomics)](https://www.spaam-community.org/).

The webinar will begin with a talk from [James A. Fellows Yates (MPI-EVA)](https://www.eva.mpg.de/archaeogenetics/staff/james-fellows-yates/) on "Improving reporting and reproducibility of lab work in biomolecular archaeology".
The presentation will be followed by a roundtable discussion on the topic of lab work reproducibility, open to all attendees.
Attendance to the webinar is free and open to the whole community, and particularly to students and early career researchers.
```
